lib/rebel/sql.rb

```diff
@@ -304,10 +308,26 @@ def equal(l, r)
       "#{name_or_value(l)} = #{name_or_value(r)}"
     end
 
+    def order(l, r)
+      "#{name(l)} #{value(raw(r.to_s))}"
+    end
+
     def assign_clause(clause)
       list(clause.map { |k, v| equal(k, v) })
     end
 
+    def order_by_clause(clause)
```

Comment:
The name should just be `by`, which will be reused for `GROUP BY` and the arg is `term`. This should return a Raw so that `by(:bar, :baz).desc` can be used. This should be able to take `name(:foo).asc` as argument. A symbol maps to a name, so if a hash is to be handled, it should be of the form `{ foo: raw('ASC') }`, not `{ foo: :asc }`.